src/lib/frontend/components/CookiesDisclaimerModal.svelte

```diff
@@ -0,0 +1,54 @@
+<script lang="ts">
+  import { cookieStore, getCookieStore } from '$lib/frontend/stores/localStorageStore/CookieStore'
+  import { onMount } from 'svelte'
+
+  let showCookiesDialog: boolean = false
+
+  onMount(() => {
```

Comment:
i don't think you need the onMount and the showCookiesDialog variable, the cookie store is a svelte store, you can get the value, subscribe to the store and set the value as well 
<img width="911" alt="image" src="https://github.com/6C-Solutions-4-Small-Businesses/online-shop-template/assets/46734336/64e4318f-6a38-473f-92d6-75c0fb0728f1">

Reply:
see https://github.com/6C-Solutions-4-Small-Businesses/online-shop-template/issues/17